src/Keyboard.h

```diff
@@ -22,81 +22,30 @@
 #ifndef KEYBOARD_h
 #define KEYBOARD_h
 
-#include "HID.h"
+// #include "HID.h"
```

Comment:
```suggestion
```
Please don't comment code. It just clutters up the source. If we need to see what came before, we can look at the commit history.